packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/dialplan-horizontal.less

```diff
@@ -0,0 +1,704 @@
+@import '../../../variables.less';
```

Comment:
Could we instead import variables from the ui-kit ? So colors and font-size are consistent

Reply:
Yes into the variables.less file it's done + other specific colors and size and font-size for VoIP components